src/pages/organism/navBar/molecules/userLocation/atoms/LocationInput.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+// constant
+import LOCATIONS from '../helper/constData';
+
+class LocationInput extends Component {
+    loadLocations = () => LOCATIONS.map(location => <option key={location.value} value={location.value}>{location.name}</option>)
```

Comment:
This should be separate atom, as this is not related to LocationInput, so it should not be created here.